### DISCUSSION TITLE
build github.com/bluenviron/mediamtx: cannot load crypto/ecdh: malformed module path "crypto/ecdh": missing dot in first path

### DISCUSSION THREAD

**Discussion by tv4you2016** [bluenviron/mediamtx#2218] [General]
go: finding github.com/gin-gonic/gin v1.9.1
go: finding github.com/pion/mdns v0.0.7
go: finding github.com/aler9/gosrt v0.0.0-20230803201410-b26769a3a222
go: finding github.com/stretchr/testify v1.8.4
go: finding github.com/pion/turn/v2 v2.1.3
go: finding github.com/gin-contrib/sse v0.1.0
go: finding github.com/benburkert/openpgp v0.0.0-20160410205803-c2471f86866c
go: finding github.com/mattn/go-isatty v0.0.19
go: finding github.com/goccy/go-json v0.10.2
go: finding github.com/json-iterator/go v1.1.12
go: finding github.com/pelletier/go-toml/v2 v2.0.8
go: finding github.com/bytedance/sonic v1.9.1
go: finding github.com/go-playground/validator/v10 v10.14.0
go: finding github.com/ugorji/go/codec v1.2.11
go: finding gopkg.in/yaml.v3 v3.0.1
go: finding google.golang.org/protobuf v1.30.0
go: finding github.com/davecgh/go-spew v1.1.1
go: finding github.com/pmezard/go-difflib v1.0.0
go: finding github.com/modern-go/concurrent v0.0.0-20180306012644-bacd9c7ef1dd
go: finding github.com/gabriel-vasile/mimetype v1.4.2
go: finding github.com/modern-go/reflect2 v1.0.2
go: finding github.com/go-playground/universal-translator v0.18.1
go: finding golang.org/x/text v0.12.0
go: finding github.com/leodido/go-urn v1.2.4
go: finding golang.org/x/arch v0.3.0
go: finding github.com/go-playground/locales v0.14.1
build github.com/bluenviron/mediamtx: cannot load crypto/ecdh: malformed module path "crypto/ecdh": missing dot in first path element

Building targets
build .: cannot find module for path .

Cleaning module download cache (golang/go#27455)

make[2]: *** [Makefile:39: /home/demo/openwrt-sdk-19.07.10-ramips-mt76x8_gcc-7.5.0_musl.Linux-x86_64/build_dir/target-mipsel_24kc_musl/mediamtx-v1.0.0/.built] Error 1
make[2]: Leaving directory '/home/demo/openwrt-sdk-19.07.10-ramips-mt76x8_gcc-7.5.0_musl.Linux-x86_64/package/mediamtx'
time: package/mediamtx/compile#9.73#6.53#19.50
make[1]: *** [package/Makefile:113: package/mediamtx/compile] Error 2
make[1]: Leaving directory '/home/demo/openwrt-sdk-19.07.10-ramips-mt76x8_gcc-7.5.0_musl.Linux-x86_64'
make: *** [/home/demo/openwrt-sdk-19.07.10-ramips-mt76x8_gcc-7.5.0_musl.Linux-x86_64/include/toplevel.mk:220: package/mediamtx/compile] Error 2

**Comment by aler9** [comment]
As written in the README, the minimum Golang version required to build the server is v1.20, which is not included in OpenWRT 19.07.10.

Upgrade OpenWRT or upgrade Go.

[a comment by an automated account is not shown]
